### PR TITLE
Configured webpack for dev enviroment

### PR DESCRIPTION
**Pull request by bombenko**

### CODE REVIEW

File: webpack.prod.config.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        new CleanWebpackPlugin(['dist']),
+        new HtmlWebpackPlugin({
+            title: 'ToDoList'
+        }),
+        new webpack.optimize.CommonsChunkPlugin({
+            name: "vendor",
+            minChunks: Infinity,
+        })
+    ],
+};
```
Comment (grachpower) [inline webpack.prod.config.js]: empty line in the end of the file

File: webpack.prod.config.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            },
+            {
+                test: /\.(woff|woff2|eot|ttf|otf)$/,
+                use: [
+                    'file-loader'
+                ]
+            },
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components)/,
```
Comment (grachpower) [inline webpack.prod.config.js]: isn't necessary to exclude bower_component. We won't use bower

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            {
+                test: /\.(woff|woff2|eot|ttf|otf)$/,
+                use: [
+                    'file-loader'
+                ]
+            },
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components)/,
+                use: {
```
Comment (grachpower) [inline webpack.config.js]: same as below

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['env']
```
Comment (grachpower) [inline webpack.config.js]: is `env` declared here? declare `env`, or use preset name

File: src/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { join } from 'lodash';
+
+function component() {
+    const element = document.createElement('div');
+
+    element.innerHTML = join(['Hello', 'webpack'], ' ');
```
Comment (grachpower) [inline src/index.js]: try to avoid doing extra work in one task commit

File: src/index.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>ToDoList</title>
+</head>
+<body>
+    <div app-root></div>
+</body>
+</html>
```
Comment (grachpower) [inline src/index.html]: empty line here
try to avoid doing extra work in one task commit

**Review by grachpower — CHANGES_REQUESTED** [review]
need work

**Review by AlenaMordvinceva — APPROVED** [review]
nravitsya vso.
tomorrow we can install ESlinter to start accustoming to the code styles.

**Review by grachpower — APPROVED** [review]

**Pull request merged**